core/src/main/java/org/kohsuke/stapler/bind/BoundObjectTable.java

```diff
@@ -75,6 +75,30 @@ public static boolean isValidJavaIdentifier(String name) {
                 .allMatch(it -> Character.isJavaIdentifierPart(it) && !Character.isIdentifierIgnorable(it) && it < 256);
     }
 
+    /**
+     * Explicitly unbind this object via HTTP.
+     *
+     * @throws HttpResponses.HttpResponseException expected outcome returning 200 OK
+     */
+    public void doRelease(StaplerRequest2 req, StaplerResponse2 rsp)
```

Comment:
`@POST`?

Reply:
I guess, although I expect this would be replaced anyway due to

> The current implementation sends one request per proxy, could be collapsed to just one if this approach is deemed preferable.